spaces/S000201/README.md

```diff
@@ -20,5 +24,5 @@ integers $n$.
 
 Not to be confused with {S139}, which has a finer topology.
 
-Defined as "infinite earring" in section 71 of {{zb:0951.54001}}.
-Appears as Example 1.25 in {{zb:1044.55001}} (available [here](https://pi.math.cornell.edu/~hatcher/AT/ATpage.html)) under the name "Shrinking wedge of circles"; however, note that it is not actually an example of a "wedge sum" as defined in e.g. {{wikipedia:Wedge_sum}}, {{doi:10.1007/978-1-4612-4576-6}}.
+Defined as "infinite earring" in section 71 of {{zb:0951.54001}}; usually known in the literature as "Hawaiian earring".
```

Comment:
This is not the right opportunity to make this change, given previous discussion on this topic.